src/NCSA.xml

```diff
@@ -0,0 +1,53 @@
+<SPDX name="University of Illinois/NCSA Open Source License" identifier="NCSA" osi-approved="true">
+  <urls>
+    <url>http://otm.illinois.edu/uiuc_openSource</url>
+    <url>http://www.opensource.org/licenses/NCSA</url>
+  </urls>
+  <license>
+    <title>
+      <p>University of Illinois/NCSA Open Source License</p>
+    </title>
+    <copyright>
+      <p>Copyright (c) 
+        <alt name="copyright" match=".+"> &lt;Year&gt; &lt;Owner Organization Name&gt;</alt>. All rights reserved. 
+      </p>
+      <p>Developed by: 
```

Comment:
should this be part of copyright tag?  (also may want to check if LLVM uses this line - if not, may want to make this line optional)